docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -6,6 +6,10 @@ Distributing collections
 
 A collection is a distribution format for Ansible content. A typical collection contains modules and other plugins that address a set of related use cases. For example, a collection might automate administering a particular database. A collection can also contain roles and playbooks.
 
+.. note::
+   Before distributing your collection, ensure you have updated the `galaxy.yml` 
+   file. See :ref:`collection_structure` for details.
```

Comment:
@samccann Hey, looks like this uses single backticks which is for interpreted text in RST. I think galaxy.yml should be wrapped in double backticks to denote a literal.

I'd also prefer to keep the entire sentence on the same line instead of orphaning the last word.

I also noticed the original PR was merged without reviews or any assigned reviewers. Can I suggest that we avoid that in future? I think the fix here is a "no backport" PR to devel and then pushing the commits to the relevant "patchback/backports" branches. Or I guess we can just merge these backports and do additional backports for the fix commit. What do you think?

Reply:
gah!  Sorry about that. Fixed it with https://github.com/ansible/ansible-documentation/pull/2192 and will backport etc from there. Gonna merge this current batch of backports now.